Tests/FitDataProtocolTests/StringExtensionsTests.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import XCTest
+@testable import FitDataProtocol
+
+final class StringExtensionsTests: XCTestCase {
+    func testDecodingAMessageString() throws {
```

Comment:
probably should also test the string from one of the other changes
`Laufen\0\0n\0\0\u{04}Èî»p\0\0\0\u{04})Ä\u{13}\u{08}"`